### PR TITLE
LIMS implementation plan template

### PR DESCRIPTION
**Pull request by idalindegaard**
### Added

- new template specific for LIMS

### Fixed

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screen shots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/lims_implementation_plan_template.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+name: LIMS Implementation plan - template
+about: template
+title: "LIMS Implementation plan"
+labels: "SWEDAC 2023/2024"
+assignees: "AnnaLeinfelt"
```
Comment (amandaraud) [inline .github/ISSUE_TEMPLATE/lims_implementation_plan_template.md:6]: @AnnaLeinfelt should you be the assignee as QA man?

**Comment by amandaraud** [comment]
> I think this looks nice! I know I said to do this in the cg_lims repo but I changed my mind and think it is more suited in validations repo. What do you say?

I agree, I think its better to keep all validations in the same repo, it will be more logical for keeping track of operations. @idalindegaard what do you think?

**Comment by idalindegaard** [comment]
> > I think this looks nice! I know I said to do this in the cg_lims repo but I changed my mind and think it is more suited in validations repo. What do you say?
> 
> I agree, I think its better to keep all validations in the same repo, it will be more logical for keeping track of operations. @idalindegaard what do you think?

I fully agree! Let's move it

**Review by amandaraud — APPROVED** [review]

**Comment by amandaraud** [comment]
Closing this PR as the Lims implementation plan has been added to the validations repo instead (see https://github.com/Clinical-Genomics/validations/pull/115)

**Pull request closed without merging**